server/controllers/conversationController.js

```diff
@@ -0,0 +1,40 @@
+const User = require("../models/User");
+const Conversation = require("../models/Conversation");
+
+// could refactor into an update call with upsert option to facilitate adding new users to existing conversations and or creating documents in one call.
```

Comment:
I think adding new users to an existing conversation is within scope of the project to do now, but we should keep in mind future extensibility in general. In this case, I think we could just add a new function for PUT requests to do that

Reply:
Will add this in next push for sure.